src/input.ts

```diff
@@ -10,6 +10,7 @@ import stringArgv from "string-argv";
 export interface Input {
     command: string;
     toolchain?: string;
+    "working-directory"?: string;
```

Comment:
`workingDirectory` to be consistent with `useCross`

Reply:
Maybe. But in all other cases of working directory usage in GH actions (and especially [run](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun)), it’s used as “working-directory”. So this is consistent with the GH action ecosystem.

Reply:
@MarcoPolo I think `working-directory` is a good name in YAML but not in JS. For example, the GitHub official action, `actions/checkout` [has input named `ssh-key`][ssh-key-in-actions] but [the name in JS is `sshKey`][name-in-js-ssh-key]. 
So I think you should use `workingDirectory` in JS as @VincentLanglet suggested.

[ssh-key-in-actions]: https://github.com/actions/checkout/blob/25a956c84d5dd820d28caab9f86b8d183aeeff3d/action.yml#L25
[name-in-js-ssh-key]: https://github.com/actions/checkout/blob/25a956c84d5dd820d28caab9f86b8d183aeeff3d/src/input-helper.ts#L112